c/authService.c

```diff
@@ -114,8 +114,7 @@ static void respond(HttpResponse *res, int rc, const ZISAuthServiceStatus
   jsonPrinter* p = respondWithJsonPrinter(res);
 
   setResponseStatus(res, HTTP_STATUS_OK, "OK");
-  setContentType(res, "application/json");
-  addStringHeader(res, "Transfer-Encoding", "chunked");
+  setDefaultJSONRESTHeaders(res);
```

Comment:
authService.c doesn't include dataservice.h. To the compiler the ```setDefaultJSONRESTHeader``` looks like ```int foo(int v)```. If compiled with LP64, the response variable and the return value will always be truncated to 4 bytes, i.e. an int.

These are the only file that have the correct function declaration:
```
./dataservice.u:dataservice.o: /u/foobar/zowe-common-c/h/dataservice.h
./zss.u:zss.o: /u/foobar/zowe-common-c/h/dataservice.h
./authService.u:authService.o: /u/foobar/zowe-common-c/h/dataservice.h
./omvsService.u:omvsService.o: /u/foobar/zowe-common-c/h/dataservice.h
./datasetService.u:datasetService.o: /u/foobar/zowe-common-c/h/dataservice.h
./zosDiscovery.u:zosDiscovery.o: /u/foobar/zowe-common-c/h/dataservice.h
```

Same applies to httpserver.c, which shouldn't include dataservice.h I think but now have to. And anyone who wants to use it will have to link with dataservice.c.